pyproject.toml

```diff
@@ -25,12 +25,13 @@ platforms = ["linux-64", "osx-64", "win-64"]
 [tool.pixi.dependencies]
 astropy = "*"
 black = "*"
-unitpackage = "*"
+unitpackage = ">=0.10.0, <0.11.0"
 isort = "*"
 mkdocs = "*"
 myst-nb = "*"
 myst-parser = "*"
 pip = "*"
+plotly = ">=5,<6"
```

Comment:
```suggestion
plotly = ">=5,<6" # Workaround for https://github.com/executablebooks/MyST-NB/issues/667
```

Reply:
The idea was that we use for linting and building the website always the latest version, and those that work with the unitpackage. Astropy is included in the unitpackage, and the module will determine the version. Aside from that, I do not see any conflicts.